README.md

```diff
@@ -1,8 +1,117 @@
-# CircularEconomy-MassFlowCalculator
+<img src="docs/images_wiki/CE-MFC.png" width="400">
+
+<table>
+<tr>
+  <td>License</td>
+  <td>
+    <a href="https://github.com/NREL/CircularEconomy-MassFlowCalculator/blob/master/LICENSE">
+    <img src="https://img.shields.io/pypi/l/pvlib.svg" alt="license" />
+    </a>
+</td>
+</tr>
+<tr>
+  <td>Build Status</td>
+  <td>
+    <a href="http://circulareconomy-massflowcalculator.readthedocs.org/en/latest/">
+    <img src="https://readthedocs.org/projects/circulareconomy-massflowcalculator/badge/?version=latest" alt="documentation build status" />
+    </a>
+  </td>
+</tr>
+<tr>
+  <td>Publications</td>
+  <td>
+    <!--- <a href="https://doi.org/10.5281/zenodo.3762635">
+    <img src="https://zenodo.org/badge/DOI/10.5281/zenodo.3762635.svg" alt="zenodo reference">
+     ---> Connected to Zenodo, awaiting first release
+    </a>
+  </td>
+</tr>
+</table>
+
+
+# CEMFC: Circular Economy - Mass Flow Calculator
```

Comment:
If this tool is to forever be specific to PV, I suggest specifying that prominently, and even consider adding PV to the name of the package. `pv_mass_flow` or something. If it might conceivably go beyond PV someday, don't bother changing the name, but still mention the specific PV use case prominently in the readme. I'd lean toward leaving the door open to beyond-PV applications.

Reply:
Renamed to PV_ICE